tests/data_io/test_connect.py

```diff
@@ -81,8 +82,12 @@ def test_get_export_config_with_args():
         "mergedArrayValuesDelimiter": ",",
         "limitRecords": 10,
         "columnsExcludeRegex": "(origin).*",
+        "exportDelta": True,
```

Comment:
Rather than modifying this test function, can you write a separate one that uses the new delta args?